processing/requirements.txt

```
@@ -1,4 +1,4 @@
-black==18.5b1
+black
```

I would use fixed versions for that kind of dependency.
```suggestion
black==18.9b0
```

---

With fixed version, we break the pre-commit. We need the pre-commit with a fixed version (but reading [here](https://github.com/ambv/black#version-control-integration), I understood that we have only two options. Use the last stable version (my change) or the master branch (it is not what we want.)
black is a dependency about code-style, so it don't affect the operation, just the code itself.

---

The way that it is today (fixed version), we will have problems like #122. Local black considers the code ok, pre-commit doesn't and fix it... forever.

---

Isn't it an option to fix `black` and `pre-commit` versions to something that actually work together?

---

I thought that there was just two options to rev: `stable` or `master`, but it worked.